aux/helpers/customer-validations/request-validation.ts

```diff
@@ -0,0 +1,59 @@
+import { Customer } from "@/domain/entities/customer-entity";
+import * as EmailValidator from 'email-validator';
+import { validateCPF } from "validations-br";
+
+export function validationRequest(body: Customer) {
```

Comment:
It can be either validateCustomerRequest or validateCustomer. This function is not general-purpose, so where else could it be called from? Maybe it should be placed in `services/customer`?